<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_ObjectSelector</class>
 <widget class="QDialog" name="W_ObjectSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>对象选择器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>选择对象</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
      <item row="1" column="0">
       <widget class="QTableWidget" name="tableWidget_page">
        <property name="styleSheet">
         <string notr="true">QTableView::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #5C87D9, stop:0.505 #557CC7, stop:1 #5075BC);
}




</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>阈值列</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>第一页 1-30</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>第二页 31-60</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTableWidget" name="tableWidget_context">
        <property name="styleSheet">
         <string notr="true">QTableView::item:selected {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #5C87D9, stop:0.505 #557CC7, stop:1 #5075BC);
}</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>id + 名称</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>1 葡萄</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>2 杏子</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>排序方式：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_sort">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>按id递增</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按id递减</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按名称递增</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>按名称递减</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_config">
       <property name="text">
        <string>选择器设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>W_ObjectSelector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
